hypersync-client/src/lib.rs

```diff
@@ -73,6 +84,8 @@ impl Client {
         })
     }
 
+    /// Retrieves blocks, transactions, traces, and logs through a stream using the provided
+    /// query and stream configuration.
```

Comment:
Can go into more detail for these. Can say it is parallelized and goes until it reaches query.to_block or server height at the time of calling the function.